editor/nodes/actions/playanimation.py

```diff
@@ -34,7 +34,7 @@ def init(self, context):
         self.add_input(NodeSocketLogicBoolean, "Stop When Done", None, {'default_value': True})
         self.add_input(NodeSocketLogicFloatFactor, "Layer Weight", None, {'default_value': 1.0})
         self.add_input(NodeSocketLogicFloatPositive, "Speed", None, {'default_value': 1.0})
-        self.add_input(NodeSocketLogicFloat, "Blendin")
```

Comment:
This is not a typo, the Logic Brick also has a "Blendin" property, it's the amount of frames the animation uses to "Blend-In"